data-validation/multiple-coordinate-schemas/generate-random-data.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

I think this is more portable